lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
Yes, this error cropped up after adding asset_host. I'll admit that I didn't dig into the 'why' behind the underlying error but I was getting a missing method error on nil object and this cleared it up.